.github/workflows/linux.yml

```diff
@@ -36,6 +36,8 @@ jobs:
     - name: build
       run: |
         make -j 2
+    - name: test
```

Comment:
could we rename these to `quicktests` or something similar? I'd prefer to disambiguate between these and the test suite.